### PR TITLE
Added make targets "install" and "uninstall"

### PR DESCRIPTION
**Pull request by ghost**
There didn't seem to be a good way to install and uninstall the tools, so I added one.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,3 +18,9 @@ ndsfuse.o: ndsfuse.c ndsrom.h tree.h
 
 clean:
 	rm -rf treetest ndstest ndsfuse *.o
+
+install:
+	cp ndsfuse ndstest treetest -t /usr/local/bin
```
Comment (magical) [inline Makefile]: I don't think ndstest and treetest need to be installed. They are just test programs, not something useful to users.

**Review by magical — CHANGES_REQUESTED** [review]
Thanks.

File: Makefile (hunk 1 above)
Comment (ghost) [reply Makefile]: Alright, then we can just get rid of them.

**Comment by ghost** [comment]
I removed the tests from the Makefile and added installation instructions.

**Review by magical — APPROVED** [review]

**Pull request merged**